src/Resources/ResourceManager/ResourceManager.csproj

```diff
@@ -12,11 +12,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="3.17.3-preview" />
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
+	<ProjectReference Include="..\Resources.Sdk\Resources.Sdk.csproj" />
+  </ItemGroup>
```

Comment:
As you add generated sdk project, why still need Microsoft.Azure.Management.ResourceManager?

Reply:
Az.Resources has cmdlets for multiple resource types owned and supported by different teams. Since we don't have ownership of the entire module nor do we have a complete understanding of the other cmdlets, updating them runs a high risk of breaking functionality for Azure customers. 
We are using the checked in SDK code with only the cmdlets that we own while the rest of the cmdlets still use the older SDK. 

Moreover, the TestFx project also uses the old SDK, making it harder to completely remove all references.